src/fatx_internal.h

```diff
@@ -46,7 +40,7 @@
 
 /* Mask to be applied when reading FAT entry values. */
 #define FATX_FAT16_ENTRY_MASK        0x0000ffff
-#define FATX_FAT32_ENTRY_MASK        0x0fffffff
```

Comment:
`0x0ffffffff` appears to have been a FAT thing? Maybe? But I do not think it applies to FATX.